string crazyflieName
string crazyflieAddress
AreaBounds localArea